test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => {
+        buf.readUIntLE(5);
+      }, RangeError
```

Comment:
@larissayvette for example, doing a regex agains the error message thrown by RangeError in this case:

``` js
assert.throws(
  () => {
    buf.readUIntLE(5);
  }, /Index out of range/
)
```